src/include/network/error_util.h

```diff
@@ -0,0 +1,135 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// error_util.h
+//
+// Identification: src/include/network/error_util.h
+//
+// Copyright (c) 2015-17, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <string>
+#include <utility>
+
+#include <event2/buffer.h>
+#include <event2/bufferevent.h>
+#include <event2/event.h>
+#include <event2/listener.h>
+
+#include "common/logger.h"
+#include "common/exception.h"
+
+namespace peloton {
+namespace network {
+/**
+ * Static utility class for some common error handling code
+ */
+class ErrorUtil {
+public:
+  ErrorUtil() = delete;
+
+  static void LogErrno() {
```

Comment:
What is this function? And why are we using `LOG_TRACE` instead of `LOG_ERROR`.

Reply:
This is migrated from clearly copy-and-pasted errno handling in the protocol handler. I suppose I can make it better as well.